### PR TITLE
Add DNS documenation, bugfix generated url, add submit line

### PR DESCRIPTION
**Pull request by szok**
- Add DNS ralph integration documentation
- Bugfix generated url to DNSAAS
- Add Ralph Submit Line form

### CODE REVIEW

File: docs/user/dns.md [hunk 1]
```diff
@@ -3,3 +3,18 @@
...
+## Configuration
+
+- ``ENABLE_DNSAAS_INTEGRATION`` - True or False if you use DNSAAS
+- ``DNSAAS_URL`` - Url to DNSAAS
+- ``DNSAAS_TOKEN`` - API Token to DNSAAS
+- ``DNS_AUTO_PTR_ALWAYS`` - DNSAAS auto_ptr value, default is 2
+- ``DNS_AUTO_PTR_NEVER`` - DNSAAS auto_ptr value, default is 1
+
+On the edit page of DataCenterAsset will apper a new tab DNS Edit.
+DNS records are searched based on DataCenterAssets IP
```
Comment (mkurek) [inline docs/user/dns.md]: are searched on -> are matched using

File: docs/user/dns.md [hunk 2]
```diff
@@ -3,3 +3,18 @@
 ## Introduction
 
 Ralph integrated with DNSAAS [PowerDNS](https://github.com/allegro/django-powerdns-dnssec)
+
+
+## Configuration
+
+- ``ENABLE_DNSAAS_INTEGRATION`` - True or False if you use DNSAAS
```
Comment (mkurek) [inline docs/user/dns.md]: set to True if you want to enable DNSaaS integration

**Comment by ar4s** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**